recipe/build.sh

```diff
@@ -4,12 +4,23 @@ set -euxo pipefail
 
 rm -rf build || true
 
+# get torch libraries for osx-arm64
+LIBTORCH_DIR=${BUILD_PREFIX}
+if [[ "$OSTYPE" == "darwin"* && $OSX_ARCH == "arm64" ]]; then
+    LIBTORCH_DIR=${RECIPE_DIR}/libtorch
+    conda list -p ${BUILD_PREFIX} > packages.txt
+    cat packages.txt
```

Comment:
Do we want to keep this in for debugging?

Reply:
That's what I figured.  It's good to have that information in the log in case we run into problems with this method.